diff --git a/src/runtime/base/program_functions.cpp b/src/runtime/base/program_functions.cpp
index 581e6c4..7759292 100644
--- a/src/runtime/base/program_functions.cpp
+++ b/src/runtime/base/program_functions.cpp
@@ -391,7 +391,7 @@ void execute_command_line_begin(int argc, char **argv, int xhprof) {
   SystemGlobals *g = (SystemGlobals *)get_global_variables();
 
   // reset global symbols to nulls or empty arrays
-  pm_php$globals$symbols_php();
+  pm_php$$globals$$symbols$_php();
 
   process_env_variables(g->GV(_ENV));
   g->GV(_ENV).set("HPHP", 1);
diff --git a/src/runtime/base/server/http_protocol.cpp b/src/runtime/base/server/http_protocol.cpp
index 6639e37..8d40178 100644
--- a/src/runtime/base/server/http_protocol.cpp
+++ b/src/runtime/base/server/http_protocol.cpp
@@ -87,7 +87,7 @@ void HttpProtocol::PrepareSystemVariables(Transport *transport,
   const VirtualHost *vhost = VirtualHost::GetCurrent();
 
   // reset global symbols to nulls or empty arrays
-  pm_php$globals$symbols_php();
+  pm_php$$globals$$symbols$_php();
 
   Variant &server = g->GV(_SERVER);
   server.set("REQUEST_START_TIME", time(NULL));
